From 40841fa0aa1a890e92d6f7763e37c26b8d0a6381 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <adrian.alonso@nxp.com>
Date: Fri, 5 Feb 2016 14:26:33 -0600
Subject: [PATCH 0933/5242] MLK-12392: arm: imx6q: fix lpddr2 build error if
 SMP is disabled
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit  05bd8100f7b66ef776c241a986977388b41af359 from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix build error if CONFIG_SMP is disabled:

arch/arm/mach-imx/busfreq_lpddr2.c: In function ‘update_lpddr2_freq_smp’:
arch/arm/mach-imx/busfreq_lpddr2.c:236:67: error: ‘me’ undeclared (first
use in this function)
  printk(KERN_DEBUG "Bus freq set to %d done! cpu=%d\n", ddr_rate, me);

Signed-off-by: Adrian Alonso <adrian.alonso@nxp.com>
Signed-off-by: Frank Li <Frank.Li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/busfreq_lpddr2.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/busfreq_lpddr2.c b/arch/arm/mach-imx/busfreq_lpddr2.c
index 8b6de08..c42b872 100644
--- a/arch/arm/mach-imx/busfreq_lpddr2.c
+++ b/arch/arm/mach-imx/busfreq_lpddr2.c
@@ -152,10 +152,10 @@ int init_mmdc_lpddr2_settings(struct platform_device *busfreq_pdev)
 int update_lpddr2_freq_smp(int ddr_rate)
 {
 	unsigned long ttbr1;
+	int me = 0;
 	int mode = get_bus_freq_mode();
 #ifdef CONFIG_SMP
 	int cpu = 0;
-	int me = 0;
 	u32 reg;
 #endif
 
-- 
1.7.9.5

